doc/source/examples/cd_clf_adult.ipynb

```diff
@@ -209,7 +209,7 @@
     "# define categorical one-hot encoder.\n",
```

Comment:
Sorry @RobertSamoilescu, one last thought before we merge, is it worth adding a note as to why we do `drop="first"` here? i.e.

```python
# Define a categorical one-hot encoder with 1st variable dropped to avoid multicollinearity
```